finagle-redis/src/main/scala/com/twitter/finagle/redis/ScriptCommands.scala

```diff
@@ -0,0 +1,147 @@
+package com.twitter.finagle.redis
+
+import com.twitter.finagle.redis.protocol._
+import com.twitter.finagle.redis.util.ReplyFormat
+import com.twitter.util.Future
+import org.jboss.netty.buffer.{ChannelBuffers, ChannelBuffer}
+
+trait Scripts { self: BaseClient =>
+  val handleUnit: PartialFunction[Reply, Future[Unit]] = {
+    case StatusReply(_) | EmptyBulkReply() => Future.Unit
+  }
+
+  val handleLong: PartialFunction[Reply, Future[Long]] = {
```

Comment:
can we make everything the end-user doesn't need to use directly `private[this]`?